transforms/dialog-full-screen-app-wrapper/dialog-full-screen-app-wrapper.js

```diff
@@ -0,0 +1,184 @@
+/*
+ * Add <AppWrapper> around the children on <DialogFullScreen>
+ * <DialogFullScreen><AppWrapper>{ children }</ AppWrapper></ DialogFullScreen>
+ */
+import registerMethods from "../registerMethods";
+
+function transformer(fileInfo, api, options) {
+  const j = api.jscodeshift;
+  registerMethods(j);
+
+  // adds the AppWrapper import path
+  const addImport = () => {
+
+    // DialogFullScreen import
+    const dialogFullScreenImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: "carbon-react/lib/components/dialog-full-screen"
+      }
+    });
+
+    const appWrapperImportString = "carbon-react/lib/components/app-wrapper";
+
+    // AppWrapper import
+    let appWrapperImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: appWrapperImportString,
+      },
+    });
+
+    // if no AppWrapper import exists already add one
+    if (!appWrapperImport.size()) {
+      appWrapperImport = j.importDeclaration(
+        [j.importDefaultSpecifier(j.identifier("AppWrapper"))],
+        j.stringLiteral(appWrapperImportString)
+      );
+    
+      // insert AppWrapper import after DialogFullScreen import
+      dialogFullScreenImport.insertAfter(appWrapperImport);
+
+      return true;
+    }
+  };
+
+  // filter the variable declarations against the expression declared as a child of DialogFullScreen
+  const filteredDeclarations = (variableDeclarations, expressionContainerName) => {
+    if (!variableDeclarations.length) {
+      return variableDeclarations;
+    }
+
+    const filteredDeclarations = [];
```

Comment:
This should be a `let` saying as you're modifying it later, I'm surprised you're not getting any errors/warnings in the editor with this

Reply:
You can push onto an const array, you just can't reassign it a value

Reply:
That seems like bad practice, I'll have a look into that

Reply:
https://stackoverflow.com/a/23436563